### ISSUE TITLE
JSON exception when adding Clean Plugin info

### ISSUE THREAD

**Issue by CptMcSplody** [loot/loot#1030]
[json.exception.type_error.302] type must be number, but is null

![image](https://user-images.githubusercontent.com/1944639/46463662-534af300-c7bc-11e8-8544-0c77d45c99cb.png)

![image](https://user-images.githubusercontent.com/1944639/46463640-43cbaa00-c7bc-11e8-865b-384e76dac88d.png)

```
[
    10: 09: 47.897019
]
[warning
]: Blocking load of resource at https: //fonts.googleapis.com/css?family=Roboto+Mono:400,700|Roboto:400,300,300italic,400italic,500,500italic,700,700italic [
    10: 10: 06.881522
]
[error
]: Failed to parse CEF query request 
"{"name":
    "editorClosed",
"editorState": {
    "applyEdits": true,
    "metadata": {
        "name": "CosmeticVampireOverhaul.esp",
        "enabled": true,
        "group": "NPC Cosmetic Overhauls",
        "after": [
            {
                "name": "Cutting Room Floor.esp",
                "display": "Cutting Room Floor.esp",
                "condition": ""
            },
            {
                "name": "Better Vampire NPCs.esp",
                "display": "",
                "condition": ""
            }
        ],
        "tag": [
            {
                "name": "NpcFaces",
                "isAddition": true,
                "condition": ""
            },
            {
                "name": "R.Head",
                "isAddition": true,
                "condition": ""
            }
        ],
        "clean": [
            {
                "crc": 1109657391,
                "itm": null,
                "udr": null,
                "nav": null,
                "util": "SSEEdit v3.2.71 EXPERIMENTAL"
            }
        ],
        "url": [
            {
                "link": "https://www.nexusmods.com/skyrimspecialedition/mods/8695/",
                "name": "Cosmetic Vampire Overhaul SSE on Nexus Mods"
            }
        ]
    }
}
}": [json.exception.type_error.302] type must be number, but is null
[
10: 10: 06.883129
]
[error
]: Chromium console message from http: //loot/ui/app.bundle.js at line 5008:
Error: [json.exception.type_error.302
] type must be number, but is null
at onFailure (http: //loot/ui/app.bundle.js:5008:5818) 
```

**Comment by WrinklyNinja** [comment]
Whoops, itm/udr/nav are optional...

**Comment by WrinklyNinja** [comment]
Fixed in a99b7fcb834be628d4eeb0527feb651e4beb18f4.
